scripts/install_texstudio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sudo apt-get update
+sudo apt install texlive-full
+
+sudo add-apt-repository ppa:sunderme/texstudio
+sudo apt remove texstudio-doc texstudio-l10n
+sudo apt update
+sudo apt install texstudio
```

Comment:
Users can use not only texstudio, script probably should download tex deps that you need to build document but you can use for this different tools like texmaker